core/include/drivers/stm32_i2c.h

```diff
@@ -120,6 +122,8 @@ struct i2c_handle_s {
 	unsigned long clock;
 	enum i2c_state_e i2c_state;
 	uint32_t i2c_err;
+	uint32_t saved_timing;
+	uint32_t saved_frequency;
```

Comment:
Should this be `unsigned long`? (just wondering since `stm32_clock_get_rate()` returns `unsigned long`)

Reply:
that's right, thanks, will fix.

Reply:
addressed in the fixup change following this change.